doc/release/platforms/README.ibm.rst

```diff
@@ -96,9 +103,9 @@ the following settings:
    site to another so check with your site's documentation for
    details.
 
-   We started work on a loadleveler launcher (see README.launcher for
+   We started work on a loadleveler launcher (see :ref:`readme-launcher` for
    a general description of the role of launchers in Chapel) which can
-   be utilized by setting the ``CHPL_LAUNCH`` environment variable to
+   be utilized by setting the ``CHPL_LAUNCHER`` environment variable to
    ``loadleveler``.  At the time of the release, this launcher was not
```

Comment:
I can update it if you really want, but there's a ton of stuff in this document that's misleading or outright wrong. I think the warning makes it obvious that this file has not been touched in a long time, so something that was just added to it was done years ago